charts/flaresolverr/Chart.yaml

```diff
@@ -2,18 +2,19 @@ apiVersion: v2
 appVersion: v1.2.3
 description: FlareSolverr is a proxy server to bypass Cloudflare protection
 name: flaresolverr
-version: 1.1.0
+version: 2.0.0
+kubeVersion: ">=1.16.0"
 keywords:
-  - flaresolverr
-  - jackett
+- flaresolverr
+- jackett
```

Comment:
is this somehow related to Jackett? maybe add the cloudflare tag instead?

Reply:
As far as I know it only works with jackett, i'm pretty sure it's maintained by the same people.